Resume.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!-- Here we import the Boostrap CSS stylesheet -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="./css/main.css"> 
+    <link rel="stylesheet" href="./css/resume.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap" rel="stylesheet">
+    <title>Responsive grid project</title>
+</head>
+
+<body>
+    <nav class="navbar navbar-expand-lg navbar-light fixed-top p-4">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="./index.html"><b>Maya Nelson</b> / PROJECT MANAGER</a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse " id="navbar">
+          <ul class="navbar-nav ms-auto">
+            <li class="nav-item">
+              <a class="nav-link active"  href="./index.html">ABOUT ME</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page" href="./Resume.html">RESUME</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active"  href="./Projects.html">PROJECTS</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active" href="./Contact.html">CONTACT</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <div id="navholder"></div>
+    <main>
+      <div>
+        <h1 class="d-flex justify-content-center mt-4 py-3 ">Resume</h1>
+        <div class="row mb-5">
+          
+            <h1 class="col-5 offset-1 col-md-4 offset-md-2 col-lg-3 offset-lg-3 col-xl-2 offset-xl-4 mb-5">Experience</h1>
+            <a href="#" class=" col-4 offset-1 col-md-3 col-lg-2  col-xl-1 btn btn-primary rounded-pill fw-bold mb-5 ">DOWNLOAD CV</a>
```

Comment:
I think you could wrap these together in the same col class, and just set `display: flex` and `space-between` on it (with your own class) to align the text to the left and the button to the right.
It's best to avoid putting `col` classes directly on buttons and images - best to apply to a div wrapping the button/image.